wgs.yaml

```diff
@@ -1,4 +1,44 @@
-sigs: []
+sigs:
+- dir: sig-feature-store
+  name: Feature Store
+  mission_statement: >
+    SIG Feature Store covers the definition, management, storage, discovery, and serving
+    of features to models.
+
+  charter_link: charter.md
+  label: area/sig-feature-store
+  leadership:
+    chairs:
+    - github: pyalex
+      name: Oleksii Moskalenko
+      company: Gojek
+    - github: woop
+      name: Willem Pienaar
+      company: Gojek
+    tech_leads:
+    - github: pyalex
+      name: Oleksii Moskalenko
+      company: Gojek
+    - github: woop
+      name: Willem Pienaar
+      company: Gojek
+  meetings:
+  - description: Monthly SIG Meeting
+    day: Wednesday
```

Comment:
so the meeting cadence is once a month?

Reply:
Yes, if that is acceptable. I would rather start slower and have more fruitful meetings.